scripts/run.js

```diff
@@ -1,27 +1,72 @@
 const fs = require("fs");
 const os = require("os");
 const path = require("path");
-const { exec, execSync } = require("child_process");
+const { exec, execSync, spawn } = require("child_process");
 
 const PORT = 5050;
+const PATH_LIMIT = 1024;
+const SPAWN_SHELL_DELAY = 1000;
+const REWRITE_NGROK_URL_DELAY = 3000;
+const NGROK_PATH = process.argv[2];
 const systemType = os.platform();
 
-if (systemType !== "darwin") {
-  console.log("This command currently only supports unix at the moment.");
-  return;
-}
+const windowsShellOptions = {
+  shell: process.env.ComSpec || "C:\\Windows\\system32\\cmd.exe",
+};
+
+const setNgrokPath = () => {
```

Comment:
I know it doesn't matter in this case, but please use a normal function instead of an arrow function, especially if it's in the global scope unless the function is passed as a parameter or within a function.

Sorry, I'll write up a coding convention for this repo later.